### PR TITLE
[gitpod] telemetry minor fixes

### PR DESCRIPTION
**Pull request by jeanp413**

### CODE REVIEW

File: src/vs/gitpod/node/gitpodInsightsAppender.ts [hunk 1]
```diff
@@ -191,7 +193,7 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 			}
 			filterEvent.add(eventName);
 			return {
-				event: `vscode_${formatEventName(eventName)}`,
+				event: 'vscode_file_access',
```
Comment (jeanp413) [inline src/vs/gitpod/node/gitpodInsightsAppender.ts:196]: should I delete `fileGet` and `filePUT` as they are no reliable?

File: src/vs/gitpod/node/gitpodInsightsAppender.ts [hunk 2]
```diff
@@ -191,7 +193,7 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 			}
 			filterEvent.add(eventName);
```
Comment (akosyakov) [inline src/vs/gitpod/node/gitpodInsightsAppender.ts:194]: I don't think it is good guard. You need to add both `editorOpened` and `fileGet`.

Maybe it is better to have explicit flags like: `readAccessTracked` and `writeAccessTracked`. If they are already true then return undefined.

File: src/vs/gitpod/node/gitpodInsightsAppender.ts (hunk 1 above)
Comment (akosyakov) [reply src/vs/gitpod/node/gitpodInsightsAppender.ts:196]: It sounds like we can drop `fileGet`, but for `filePut` we don't have alternative, so let's keep it?

File: src/vs/server/remoteExtensionHostAgentServer.ts [hunk 3]
```diff
@@ -298,7 +298,7 @@ export class RemoteExtensionHostAgentServer extends Disposable {
 				this._register(toDisposable(() => appInsightsAppender!.flush())); // Ensure the AI appender is disposed so that it flushes remaining data
 			}
 
-			appInsightsAppender = new GitpodInsightsAppender();
+			appInsightsAppender = new GitpodInsightsAppender(this._productService.nameLong, this._productService.version + '-remote');
```
Comment (akosyakov) [inline src/vs/server/remoteExtensionHostAgentServer.ts:301]: just version without `-remote` should work

Comment (akosyakov) [inline src/vs/server/remoteExtensionHostAgentServer.ts:301]: `nameShort` should be enough? i.e. `VS Code` not `Visual Studio Code`

File: src/vs/gitpod/node/gitpodInsightsAppender.ts (hunk 2 above)
Comment (jeanp413) [reply src/vs/gitpod/node/gitpodInsightsAppender.ts]: done

File: src/vs/gitpod/node/gitpodInsightsAppender.ts (hunk 1 above)
Comment (jeanp413) [reply src/vs/gitpod/node/gitpodInsightsAppender.ts:193]: removed `fileGet`

**Review by akosyakov — APPROVED** [review]
LGTM ❤️

**Pull request merged**